core/string/ustring.cpp

```diff
@@ -3577,6 +3577,35 @@ String String::dedent() const {
 	return new_string;
 }
 
+String String::strip_bbcode() const {
+	String new_string;
+	for (int i = 0; i < length(); i++) {
+		bool found_close = false;
+		// Checks for "["
+		if (operator[](i) == 91) {
```

Comment:
No need to use numbers instead of characters:

```suggestion
		if (operator[](i) == '[') {
```